models/ShoppingCart.js

```diff
@@ -0,0 +1,34 @@
+const { Model, DataTypes } = require("sequelize");
+const sequelize = require("../config/connection");
+
+class shopping_cart extends Model {}
+
+ShoppingCart.init({
+  id: {
+    type: DataTypes.INTEGER,
+    allowNull: false,
+    primaryKey: true,
+    autoIncrement: true,
+  },
+  order_date: {
+    type: DataTypes.STRING,
+    allowNull: false,
+  },
+  total_amount: {
+    type: DataTypes.INTEGER,
+    allowNull: false,
+    primaryKey: true,
+    autoIncrement: true,
+  },
+  order_number: {
+    type: DataTypes.STRING,
+    allowNull: false,
+  },
```

Comment:
I think you're missing a foreign key column here so that we can tell which customer is buying the Shopping Cart. You can take a look at the recipe.category_id column for an example of a foreign key column.